src/uk/gov/hmcts/contino/TeamNames.groovy

```diff
@@ -23,10 +23,10 @@ class TeamNames {
                       'ccpay':'Fees/Pay',
                       'bar':'Fees/Pay',
                       'probate':'Probate',
-                      'bulk-scan':'Software Engineering',
-                      'rpe':'Software Engineering',
-                      'draft-store':'Software Engineering',
-                      'pbi':'Software Engineering',
+                      'bulk-scan':'RPE',
+                      'rpe':'RPE',
```

Comment:
Software Engineering is used for calculating billing, I think this needs a second level of config, i.e. `namespace` would be good to also have support for things like default slack channel etc....